tasks.py

```diff
@@ -859,6 +860,25 @@ def unittest_coverage(context):
     run_command(context, command)
 
 
+@task(pre=[unittest])
+def unittest_xml_coverage(context):
+    """Produce an XML coverage report that can be ingested into other tools."""
+    produce_xml_cmd = "coverage xml -o coverage.xml"
+    run_command(context, produce_xml_cmd)
+
+    # Since the container runs from a different working dir,
+    # We have to edit the XML to inject the current directory
+    # to play nice with tools outside of the container.
+    cur_dir = os.path.dirname(__file__)
+    cov_xml_path = os.path.join(cur_dir, "coverage.xml")
+
+    tree = ET.parse(cov_xml_path)  # noqa: S314
+    root = tree.getroot()
+
+    root.find(".//sources/source").text = cur_dir
+    tree.write(cov_xml_path, encoding="utf-8", xml_declaration=True)
```

Comment:
I noticed that this was removed from a related PR into Cookiecutter-nautobot-app, is the Etree going to be removed here too?